modin/data_management/data_manager.py

```diff
@@ -122,9 +122,11 @@ def pandas_index_extraction(df, axis):
     def _prepare_method(self, pandas_func, **kwargs):
         """Prepares methods given various metadata.
 
-        :param pandas_func:
-        :param kwargs:
-        :return:
+        Args:
+            pandas_func: The function to prepare
+
+        Returns
+            helper function which handles potential transpose
```

Comment:
Let's also use upper case for the first letter and add a period to the end of the return statement.

There are more of these below.